pkg/option/config.go

```diff
@@ -1119,6 +1119,10 @@ const (
 	// EnableStaleCiliumEndpointCleanup sets whether Cilium should perform cleanup of
 	// stale CiliumEndpoints during init.
 	EnableStaleCiliumEndpointCleanup = "enable-stale-cilium-endpoint-cleanup"
+
+	// CustomResourceUpdateRate is the maximum rate in which a custom
+	// resource is updated
+	CustomResourceUpdateRate = "custom-resource-update-rate"
```

Comment:
I wonder if we should name this a bit more specifically. There are lots of different custom resources the operator manages or updates for various purposes, and the CiliumNode CRD is just one of them.

I think we should name it `ipam-cilium-node-update-rate` or something similar (that implies both a relationship to IPAM and CiliumNode) instead.